enterprise/azuredeploy.json

```diff
@@ -60,6 +60,42 @@
         "metadata": {
           "description": "Your ssh public key."
         }
+      },
+      "vaultResourceGroup": {
```

Comment:
As discussed, let's put the old parameters file, the old run.sh and the old azuredeploy.json in s separate directory called withoutTLS so people can use that if they want.